processor/redactionprocessor/README.md

```diff
@@ -102,6 +102,20 @@ processors:
     # - `info` includes just the redacted key counts in the summary
     # - `silent` omits the summary attributes
     summary: debug
+    # url_sanitization configures URL sanitization to remove variable elements from the url, causing high cardinality issues
+    url_sanitization:
+      # enabled controls whether URL sanitization is active
+      enabled: true
+      # attributes is a list of attribute keys that contain URLs to be sanitized
+      attributes: ["http.url", "url"]
+      # max_segments is the maximum number of path segments to retain in the URL
+      max_segments: 4
+      # cache_size is used to cache valid elements from the url
+      cache_size: 1000
+      # replace_with is the string used to replace sanitized segments in the URL
+      replace_with: "***"
+      # sanitize_span_name controls whether to apply URL sanitization to span names
+      sanitize_span_name: true
```

Comment:
This configuration interface seems pretty inconsistent with the rest of the redaction processor. Can we simplify it? E.g. why do we have `replace_with` specifically for `url_sanitization` while the rest of the processor's functionality doesn't have this configurable?

Also why do we need to bring another dependency? It seems like we are forced to use the interface provided by that library. Can we take only the necessary code and host it in the processor?